frontend/src/helpers/renderHelpers.tsx

```diff
@@ -0,0 +1,13 @@
+import { useTypedSelector } from '../redux/reducers/reducers';
+
+export default function DetailsViewRenderMapper(props: { resource: any }) {
+  const { resource } = props;
+  const detailViews = useTypedSelector(state => state.ui.views.details.pluginAppendedDetailViews);
+  return (
+    <>
+      {detailViews.map((item, index) => {
+        return <item.renderComponent resource={resource} key={`${item.sectionName}__${index}`} />;
```

Comment:
This needs to be added to a SectionBox, and it's title also added as the title. That's why I mentioned we should move the title out of the plugin registration and into the return of the sectionFunc.